docs/source/daml2ts/index.rst

```diff
@@ -21,7 +21,7 @@ Usage
 
 ``daml2ts`` is invoked via the DAML SDK assistant.
 
-In outline, the command to generate TypeScript from DAML is ``daml codegen ts DAR -o OUTDIR –main-package-name=PACKAGE`` where ``DAR`` is the path to a DAR file (generated via ``daml build``), ``OUTDIR`` is a directory where you want the TypeScript to be written and ``PACKAGE`` is a desired TypeScript package name.
+In outline, the command to generate TypeScript from DAML is ``daml codegen ts DAR -o OUTDIR`` where ``DAR`` is the path to a DAR file (generated via ``daml build``), ``OUTDIR`` is a directory where you want the TypeScript to be written and ``PACKAGE`` is a desired TypeScript package name.
```

Comment:
There is no more `PACKAGE` parameter and we're finally writing JavaScript + typing. We should embrace that:
```suggestion
In outline, the command to generate JavaScript and TypeScript typings from DAML is ``daml codegen ts DAR -o OUTDIR`` where ``DAR`` is the path to a DAR file (generated via ``daml build``) and ``OUTDIR`` is a directory where you want the JavaScript to be written.
```
Can you please check there are no more references to `PACKAGE` elsewhere.

Reply:
(1) Fixed.
(2) Done.